templates/partials/answers/currency.html

```diff
@@ -8,6 +8,7 @@
   "name": answer.id,
   "value": input._value() | e,
   "label": {
+    "id": answer.id + "-label",
```

Comment:
We should use the recommended way to concatenate strings which is using `~` so: `answer.id ~ "-label"`.